documentation/process.md

```diff
@@ -35,12 +35,19 @@ A "normative" change is change to the specification that is not editorial, that
     - Any working group member can add themselves as a reviewer.
     - Any one from the general public is allowed to review PRs in order to inform the ARIA WG of approval or concerns.
 3. **Tests:** If the change is testable, tests should be add to the validators directory or WPT repo before merging, or, if the PR owner does not have the expertise to write tests, issues should be file in the ARIA repository as a follow up. See [Aria Test Overview](tests.md).
-4. **APG:** If the change requires a change to the [APG](https://github.com/w3c/aria-practices), an issue on the APG should be made describing the change.
-5. **Implementation:** If the change requires implementation changes, issues should be opened on the browser after the PR has been approved.
-   - At least one implementation is required for merge. Implementation or implementation commitment from two browsers is required for merge.
+4. **APG:** If the PR will required changes to the [APG](https://github.com/w3c/aria-practices), an issue on the APG should be made describing the change.
+5. **ACT Review:** If the PR will require [ACT rule](https://www.w3.org/WAI/standards-guidelines/act/rules/) changes, then the label "needs ACT review" must be added. This will trigger the process of triage and potentially review by the [ACT Task Force](https://www.w3.org/WAI/about/groups/task-forces/conformance-testing/). When the PR has been reviewed and concerns resolved, the ACT task force will mark the PR as approved by checking the box in the PR tracking list in the description of the PR. The tag may be added at different times:
```

Comment:
```suggestion
5. **ACT Review:** If the PR will require [ACT rules](https://www.w3.org/WAI/standards-guidelines/act/rules/) changes, then the label "needs ACT review" must be added. This will trigger the process of triage and potentially review by the [ACT Task Force](https://www.w3.org/WAI/about/groups/task-forces/conformance-testing/). When the PR has been reviewed and concerns resolved, the ACT Task Force will mark the PR as approved by checking the box in the PR tracking list in the description of the PR. The tag may be added at different times:
```